go/exocom/main_test.go

```diff
@@ -28,6 +29,8 @@ func FeatureContext(s *godog.Suite) {
 	var shouldCmdBeKilled bool
 	var server *http.Server
 	var exocomPort int
+	var lastSentMessage string
+	var lastSentMessageName string
```

Comment:
What are these variable used for? They appear to be assigned but are otherwise unused